apps/settings/elements/developer_hud.html

```diff
@@ -69,6 +69,16 @@ <h2 data-l10n-id="hud-performance"></h2>
           </gaia-checkbox>
         </li>
         <li>
+          <p data-l10n-id="reflow-threshold"></p>
+          <span class="button icon icon-dialog">
+            <select name="devtools.reflowduration.threshold" data-value-type="integer">
+              <option value="1" selected>No Threshold</option>
+              <option value="16">16</option>
```

Comment:
After thinking some more about it, 16ms is indeed the length of a frame at 60 FPS, but if your reflow takes the full 16ms, it might not leave enough time for things like JS and repaint to push the frame out in time (i.e. a 15ms reflow will probably cause a frame drop).

10ms seemed like a good value to me. What about the following thresholds: 0, 10, 20, 50, 100? I think that eventually, we'd like the `jank-threshold` to have the same values, but no need to change that now.